### PR TITLE
Removed OOOForm data after submitting and UI change

### PR DESCRIPTION
**Pull request by dhruv036**
**Issue: #321** 

**Disciption:**
1. Before after submitting OOOForm desciption of saved in state and when we clicked for new submission it's shows previous  form data.

**Recording:**

https://github.com/Real-Dev-Squad/mobile-app/assets/56362412/ce274ec1-5a68-41ea-95f2-0dc820b5c891

### CODE REVIEW

File: android/gradle.properties [hunk 1]
```diff
@@ -24,5 +24,8 @@ android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
 
+org.gradle.jvmargs=-Xmx4096m
```
Comment (shreya-mishra) [inline android/gradle.properties]: why this change ?

File: android/gradle.properties [hunk 2]
```diff
@@ -24,5 +24,8 @@ android.useAndroidX=true
...
+
+
 # Version of flipper SDK to use with React Native
 FLIPPER_VERSION=0.99.0
```
Comment (shreya-mishra) [inline android/gradle.properties]: why this change ?

File: src/screens/HomeScreen/HomeScreenV2.tsx [hunk 3]
```diff
@@ -57,6 +57,9 @@ const HomeScreenV2 = (): JSX.Element => {
     };
     const res = await submitOOOForm(data, loggedInUserData?.token);
     console.log(res);
```
Comment (shreya-mishra) [inline src/screens/HomeScreen/HomeScreenV2.tsx]: please remove this log

File: android/gradle.properties (hunk 2 above)
Comment (dhruv036) [reply android/gradle.properties]: Need more heap storage for gradle build so that's why added.

File: android/gradle.properties [hunk 4]
```diff
@@ -24,5 +24,6 @@ android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
 
+
```
Comment (shreya-mishra) [inline android/gradle.properties]: please remove this line

**Review by shreya-mishra — APPROVED** [review]
